scss/_popover.scss

```
@@ -30,7 +34,7 @@
   // So reset our font and text properties to avoid inheriting weird values.
   @include reset-text();
   @include font-size(var(--#{$prefix}popover-font-size));
-  font-weight: var(--#{$prefix}popover-font-weight); // Boosted mod
+  line-height: var(--#{$prefix}popover-body-line-height); // Boosted mod
```

```suggestion
  line-height: var(--#{$prefix}popover-line-height); // Boosted mod
```
Not sure about this change since the line-height is the one used for the body, you tell me !

---

It makes more sens I think so yes we can take it IMO ;)